physionet-django/user/test_integration.py

```diff
@@ -211,7 +211,7 @@ def test_credential_application(self):
             'application-reference_email': 'root@example.com',
             'application-reference_organization': 'MIT',
             'application-reference_title': 'Administrator',
-            'application-research_summary': 'Effects of asdfghjk on zxcvbnm',
+            'application-research_summary': 'I am a researcher ' * 5,
```

Comment:
Please could we add a full string here with the kind of comment you would like to see?